lib/config.ps1

```diff
@@ -21,13 +21,14 @@
     "computer"
     # "custom_time"  # use custom info line
     "uptime"
-    "pkgs"
+    "resolution"
```

Comment:
Resolution should probably be commented out by default, most people know their resolution and we don't want to give the user to much information at the beginning

Reply:
I was assuming the default config list was taking the fastest operations in mind. Resolution part is just reading an environment variable, so I put it in.

Reply:
That makes sense, but the default config should ideally be the most popular information that most people would likely want enabled, then they can decide to remove info in favour of speed, or vice-versa.